packaging/nnstreamer.spec

```diff
@@ -370,7 +386,8 @@ NNStreamer's tensor_fliter subplugin of caffe2
 
 %package devel
 Summary:	Development package for custom tensor operator developers (tensor_filter/custom)
-Requires:	nnstreamer = %{version}-%{release}
+Requires:	nnstreamer-core = %{version}-%{release}
+Conflicts: nnstreamer-configuration
```

Comment:
Does this mean that we cannot install the ```nnstreamer-devel``` package after installing the ```nnstreamer``` package? Is it right?

Reply:
This is need to enable envvars in another repo which is developing a nnstreamer sub-plugin (causing nntrainer inference plugin unittest fail). But putting conflicts here would create some side-effect.
@jaeyun-jung  and I discussed offline about this matter, Conflicting with `configuration` is related to sub-plugin test, so moved this to `test-devel`

Reply:
Hmm. In that case. it looks OK.